.circleci/config.yml

```diff
@@ -1,41 +1,93 @@
 version: 2
+
+aliases:
+  restore_cache: &restore_cache
+    restore_cache:
+      name: Restore node_modules cache
+      keys:
+        - npm-cache-{{ checksum "package-lock.json" }}
+
+  persist_cache: &persist_cache
+    save_cache:
+      name: Save node modules cache
+      key: npm-cache-{{ checksum "package-lock.json" }}
+      paths:
+        - node_modules
```

Comment:
My understanding is that there is no need to cache node_modules when using `npm ci`

Reply:
You're right, documentation states:

> If a node_modules is already present, it will be automatically removed before npm ci begins its install.